auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -1,24 +1,40 @@
 <template>
   <v-tooltip top max-width="450px" light content-class="tooltip">
     <template v-slot:activator="{ on }">
-      <span v-on="on" class="tooltip-text">Numbered Benefit Company</span>
+      <span v-on="on" class="tooltip-text" v-if="enableBcCccUlc">Numbered Company</span>
+      <span v-on="on" class="tooltip-text" v-else>Numbered Benefit Company</span>
     </template>
     <v-card class="tooltip-content">
-      <h3 class="mb-3">Numbered Benefit Company</h3>
-      <span>A Benefit Company can choose to use as its name the incorporation number of the company followed by “B.C.
-        Ltd.” The incorporation number is assigned by the Business Registry after the Incorporation Application is filed
-        and the company is incorporated.</span>
+      <div v-if="enableBcCccUlc">
```

Comment:
No need to change but this could have been a `template` element, which won't get rendered, whereas this `div` is rendered. (Sometimes you need a div for flex but probably not here.)

Don't change unless you're changing something else. This is low priority.

Reply:
This comment applies to the other Vue files if you do make an update.

Reply:
Done